scripts/keycloak_aws_instance.sh

```
@@ -0,0 +1,771 @@
+        echo -e "   ${GREEN}OK${NC} VPC deleted" ||
+        echo -e "   ${YELLOW}Warning: Could not delete VPC (may have remaining dependencies)${NC}"
+}
+
+destroy_instance() {
+    echo ""
+    echo -e "${RED}Destroying Keycloak Instance${NC}"
+    echo -e "${RED}============================${NC}"
+    echo ""
+
+    if [ -f "$INFO_FILE" ]; then
+        source "$INFO_FILE"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)